README.md

```diff
@@ -1,81 +1,217 @@
 # druid
 
-## Data-oriented Rust User Interface Design toolkit
+## A data first Rust-native UI toolkit.
 
-Druid is a new Rust-native UI toolkit, still in early stages. Its main
-goal is performance, also aiming for small binary size and compile time,
+[![crates.io](https://meritbadge.herokuapp.com/druid)](https://crates.io/crates/druid)
+[![docs.rs](https://docs.rs/druid/badge.svg)](https://docs.rs/druid/)
+[![license](https://img.shields.io/crates/l/druid)](./LICENSE)
+
+TKTK REVISIT GOALS? ... Its main goal is performance, also aiming for small binary size and compile time,
 fast startup, and very easy build configuration (just `cargo run`).
 It currently supports Windows, macOS, and GNU/Linux.
 
-Raph gave a talk at the July 2018 SF Rust Meetup ([video][jul-2018-video],
-[slides][jul-2018-slides]) about the design. Traditional object-oriented
-designs have proved to be cumbersome.
+It is currently used as a basis for [Runebender], a font editor written in Rust.
```

Comment:
Druid's current development is largely driven by its use in [Runebender], a new font editor.